### PR TITLE
Complete update of /support/plans-and-pricing page

### LINKED ISSUE #2198: Update plans and pricing page
https://docs.google.com/document/d/1sTSMTDUD6yfgrbj-VazqvQw77gA4Gf1s8p3LglCYITM/edit?ts=599ef989

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Complete content re-write of the /support/plans-and-pricing page. Separated the price tables out into re-usable includes.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Check visually and against the [copydoc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#heading=h.e7s406oqmgi9)

## Issue / Card

Fixes #2198

### CODE REVIEW

File: templates/support/plans-and-pricing.html [hunk 1]
```diff
@@ -6,298 +6,225 @@
...
+
+  <div class="row">
+    <div class="col-10" style="overflow-x: auto;">
+      {% include "shared/pricing/_maas-support.html" %}
+    </div>
+  </div>
+
+  <div class="row">
+    Install MAAS [https://www.ubuntu.com/download/server/provisioning]
+    Contact us [https://www.ubuntu.com/server/maas/contact-us]
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: These should be links.

File: templates/support/plans-and-pricing.html [hunk 2]
```diff
@@ -6,298 +6,225 @@
...
   </div>
 </section>
-<section class="p-strip--x-light is-deep is-bordered">
+
+<section class="p-strip--x-light is-bordered" id="kubernetes">
+  <div class="row">
+    <div class="col-8">
+      <h2>Enterprise support for Kubernetes</h2>
+      <p>Scale-out container management is top of mind for savvy cloud architects around the world.  Ubuntu is, by far, the world&rsaquo;s leading platform for container-based workflows.  Canonical provides enterprise-caliber support for Kubernetes.</p>
+      <p>For more information about Kubernetes support, <a href="/containers/contact-us">contact us</a></p>
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: Chevron?

File: templates/shared/pricing/_storage-support.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+    <tr>
+      <td>Production grade monitoring</td>
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+  </tbody>
+</table>
+
+<p><a href="https://insights.ubuntu.com/2015/05/18/ubuntu-advantage-storage-factsheet/?_ga=2.122490379.1666167933.1504515501-463981611.1499434659" class="p-link--external">Download the datasheet</a> or, <a href="/containers/contact-us?product=containers-kubernetes-foundation">contact us&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/shared/pricing/_storage-support.html]: I prefer the contact us been a button and the datasheet link is a link. Like the section above.

File: templates/support/plans-and-pricing.html [hunk 4]
```diff
@@ -6,298 +6,225 @@
...
   <div class="row">
-    <div class="col-10">
-      <h3>Service provider</h3>
-      <p>For large volume and public clouds.</p>
-      {% include "shared/_ubuntu_advantage_cloud_service_provider.html" %}
+    <div class="col-8">
+      <h2>Enterprise support for MAAS</h2>
+      <p>MAAS is freely available, open source software from Canonical. Many enterprises depend on MAAS to automate and optimize provisioning for production hardware.  Canonical offers Standard and Advanced support plans priced per-machine-per-month or, for large volume and public clouds with dynamic scale, we offer per-region pricing with flexible node ranges.</p>
     </div>
+  </row>
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: `</div>`

File: templates/support/plans-and-pricing.html [hunk 5]
```diff
@@ -6,298 +6,225 @@
...
+
+  <div class="row">
+    <div class="col-10" style="overflow-x: auto;">
+      {% include "shared/pricing/_maas-support.html" %}
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: The is `<th>` here that needs a closing tag.

File: templates/shared/pricing/_openstack-support.html [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+          <td class="u-align--center">$5,450/year</td>
+        </tr>
+        <tr>
+          <td>Or, price per region **</td>
+          <td>-</td>
+          <td>Small: $95,000/year<br />Medium: $225,000/year<br />Large: $420,000/year</td>
+          <td>-</td>
+        </tr>
+        <tr>
+          <td>Industry-leading cloud operations tooling</br>(Ubuntu, MAAS, Juju, LXD)</td>
```
Comment (anthonydillon) [inline templates/shared/pricing/_openstack-support.html]: This should be `<br />`

File: templates/shared/pricing/_storage-support.html [hunk 7]
```diff
@@ -0,0 +1,54 @@
+<table>
+  <thead>
+    <td>&nbsp;</td>
```
Comment (anthonydillon) [inline templates/shared/pricing/_storage-support.html]: Missing `<tr>`

**Comment by anthonydillon** [comment]
QA issues:
- Ubuntu Advantage for servers table is not scrollable on small screens.
- Enterprise support for OpenStack table is not scrollable on small screens.
- Enterprise support for Kubernetes table is not scrollable on small screens.
- Kubernetes consulting packages on small screens the button and link at the bottom of the section are to tight

**Comment by deadlight** [comment]
@anthonydillon requested changes made, demo updated

@frankiegranato [demo](http://www.ubuntu.com-pr-2223.run.demo.haus/support/plans-and-pricing)

File: templates/shared/pricing/_openstack-support.html [hunk 8]
```diff
@@ -0,0 +1,91 @@
+<div class="row" style="overflow-x: scroll;">
```
Comment (anthonydillon) [inline templates/shared/pricing/_openstack-support.html]: Should be `overflow-x: auto` so it only shows scrollbars when needed.

File: templates/shared/pricing/_storage-support.html [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+    <tr>
+      <td>Production grade monitoring</td>
+      <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
+    </tr>
+  </tbody>
+</table>
+
+<p><a href="/containers/contact-us?product=containers-kubernetes-foundation" class="p-link--external">Contact us</a> or <a href="https://insights.ubuntu.com/2015/05/18/ubuntu-advantage-storage-factsheet/?_ga=2.122490379.1666167933.1504515501-463981611.1499434659">download the datasheet&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/shared/pricing/_storage-support.html]: External link should be on the datasheet not the contact us.

**Comment by pmahnke** [comment]
Really close... just three comments.

- [ ] more space above 'Ubuntu Advantage for servers'
- [ ] more space above 'Optional add ons'
- [ ] most strips could use 'is-deep' I think

**Comment by anthonydillon** [comment]
@pmahnke Thanks, fixed up the page.

**Pull request merged**